### ISSUE TITLE
File colors appear later; last changed notes do not appear anymore

### ISSUE THREAD

**Issue by merlinuwe** [tgrosinger/recent-files-obsidian#99]
Thanks for your great plugin which I appreciate since years.

If I click the button, the colors of the files appear some time (~1 min.) later.

![image](https://github.com/user-attachments/assets/e7e92661-b334-431c-8463-463973a1c0ed)

![image](https://github.com/user-attachments/assets/4b4a42e7-557c-4f9c-9991-d36f71953686)

---

The list also does not show the last changed notes anymore.

(Someone confirmed this in reddit a few days ago.)

SYSTEM INFO:
	Obsidian version: v1.7.4
	Installer version: v1.7.4
	Operating system: Windows 10 Home 10.0.19045
	Login status: not logged in
	Language: de
	Insider build toggle: off
	Live preview: off
	Base theme: dark
	Community theme: MagicUser v20.12.0
	Snippets enabled: 16
	Restricted mode: off
	Plugins installed: 197
	Plugins enabled: 27
		1: Natural Language Dates v0.6.2
		2: Navigate Cursor History v1.1.0
		3: Remember cursor position v1.0.9
		4: Reset Font Size v1.0.2
		5: Set View Mode per Note v1.0.3
		6: Beautitab v1.6.1
		7: Dataview v0.5.67
		8: Dictionary v2.22.0
		9: Editing Toolbar v2.4.16
		10: Image in Editor v2.2.6
		11: Linter v1.27.1
		12: Omnisearch v1.25.0
		13: Status Bar Organizer v2.1.2
		14: Style Settings v1.0.9
		15: Supercharged Links v0.12.1
		16: Templater v2.9.0
		17: Typing Transformer v0.4.0
		18: Recent Files v1.7.0
		19: Better Export PDF v1.9.2
		20: Number Headings v1.16.0
		21: Quick Explorer v0.2.10
		22: Paste image rename v1.6.1
		23: Enhanced Symbols Prettifier v1.4.1
		24: Auto Link Title v1.5.4
		25: Plugins Annotations v1.7.7
		26: Custom Classes v2.6.1
		27: Advanced Tables v0.22.1

**Comment by tgrosinger** [comment]
This plugin does not provide file coloration. Can you tell me a little more about where those colors are coming from?

**Comment by zayKenyon** [comment]
Could this be related to https://github.com/mdelobelle/obsidian_supercharged_links/issues/155 ? I'm not sure whether this is a Recent Files or Supercharged Links issue.

**Comment by merlinuwe** [comment]
Thanks for your investigation, but supercharged links plugin hasn't changed since months and the problem, that the color of the file doesn't show in the recent files plugins list was after that.

**Comment by tgrosinger** [comment]
At this time I don't have the capacity to investigate how other plugins function and their interaction with this plugin, but I welcome and will review pull requests if someone else wants to take this on.

**Comment by merlinuwe** [comment]
Thank you very much. 

BTW: The problem is the same with Obsidian release 1.7.6.

**Comment by merlinuwe** [comment]
It seems that with the latest release 1.7.2 of your plugin both mentioned issues are solved:

- I see the colored entries

- I also see the last changed notes in the list.

![image](https://github.com/user-attachments/assets/c75dab0c-f3aa-4321-aeaa-0875d8ab6fe5)

I restarted Obsidian 3 times and I immediately see the colors and the latest changed files. I'll keep an eye on it for a while, but I think the problem has been solved permanently. 

Thank you very much!
